apps/src/code-studio/components/progress/SectionSelector.jsx

```diff
@@ -52,6 +52,7 @@ const SectionSelector = React.createClass({
     const { requireSelection, sections, selectedSectionId } = this.props;
     return (
       <select
+        className="uitest-sectionselect"
```

Comment:
Make it clear that this is just used by UI tests, and not used for things like styling (or to modify the dom from outside of this component). We do this in a few places.